plugin/src/utils/cache.ts

```diff
@@ -134,11 +155,11 @@ export const getHardCachedData = async ({ key }) => {
 
   const data = await hardCache.get(key)
 
-  return data
+  return data as T
 }
 
-export const getHardCachedNodes = async () => {
-  const allWpNodes = await getHardCachedData({ key: `allWpNodes` })
+export const getHardCachedNodes = async (): Promise<null | Node[]> => {
+  const allWpNodes = await getHardCachedData<Node[]>({ key: `allWpNodes` })
```

Comment:
Same as above :)

Reply:
yup! thats why i provided the type arg, since in this case it's returning nodes. you can provide `getHardCachedData<string>` or whatever you expect